src/pages/cli/graphql/client-code-generation.mdx

```diff
@@ -21,6 +21,8 @@ You’ll see questions as before, but now it will also automatically ask you if
 
 When you deploy your GraphQL API to the cloud, you are prompted to configure codegen. When a project is configured to generate code with codegen, it stores all the configuration `.graphqlconfig.yml` file in the root folder of your project.
```

Comment:
We should explicitly state that customers should not modify this file if we're calling it out.

Reply:
Updated